stub/src/main/java/io/grpc/stub/ServerCalls.java

```diff
@@ -349,4 +349,31 @@ public void onCancel() {
     public void onComplete() {
     }
   }
+
+  public static void unimplementedStreamResponse(StreamObserver responseObserver) {
```

Comment:
I agree with @ejona86. Helper functions lets us change functionality of given func without changes to compiler and recompiling protos on user side - that's a big win imo.

I've added two separate methods `asyncUnimplementedUnaryCall` and `asyncUnimplementedStreamingCall` as suggested by Eric.